src/datatypes/dataset.ts

```diff
@@ -193,9 +194,9 @@ export interface Formatter extends Lookup<Function | undefined> {
   'DATASET'?: (v: Dataset) => string;
 }
 
-let defaultFormatter: Formatter = {
+const DEFAULT_FORMATTER: Formatter = {
   'NULL': (v: any) => 'NULL',
-  'TIME': (v: Date) => v.toISOString(),
+  'TIME': (v: Date, tz: Timezone = Timezone.UTC) => moment.tz(v, tz.toString()).format(),
```

Comment:
I think timezone should default to UTC in the `toTabular` (the formatter should do minimal work!)